app/models/case_study.rb

```diff
@@ -9,6 +9,8 @@ class CaseStudy < Edition
   include Edition::WorldwideOrganisations
   include Edition::LeadImage
 
+  after_update :update_lead_image, if: :saved_change_to_image_display_option?
```

Comment:
it feels slightly easier to do this here rather then munge the params in the controller action to set the lead_image_id based on the hidden field via nested params.

It also allows the field to be updated on the edition even if it's invalid which unfortunately is a requirement as you should be able to create new images and set a new lead image even if the change note has not been filled in.

Reply:
Also it felt like the right thing to do to set the lead_image to nil once the user selects to hide a lead image, rather than keep it and override it via the `image_display_option`. Otherwise the case study still has a lead image, associated with it. 

Just checking that sounds right to other people?